packages/contentful-cms-loader/src/index.ts

```diff
@@ -31,7 +31,7 @@ const refByOptions: Options<RefByKey, any, string> = {
 
 const isFulFilled = <T>(p: PromiseSettledResult<T>): p is PromiseFulfilledResult<T> => p.status === 'fulfilled';
 
-const createLoaders = (config: LastRevAppConfig, defaultLocale: string): ContentfulLoaders => {
+const createLoaders = (config: LastRevAppConfig, defaultLocale: string): CmsLoaders => {
```

Comment:
This loader is specific to contentful, but I don't think one was created for Sanity